branding/EULA

```diff
@@ -1,8 +1,8 @@
 The programs included with the XCP-ng system are free software.
-To know more about the licenses of all programs,
-please read each COPYING file in the /usr/share/doc/* folders.
+
+To know more about the licenses of all programs, please read each COPYING file in the /usr/share/doc/* folders.
 
 XCP-ng comes with no warranty, to the extent permitted by applicable law.
 
-For more info on XCP-ng, please check https://xcp-ng.org
-Pro support for production environment is available on https://xcp-ng.com
+For more info on XCP-ng, please check https://xcp-ng.org/.
```

Comment:
I'm not fond of using a trailing slash, any reason for that? (best practice?)

Also, I find it unpleasant to have a dot at the end of a link (and even worse, after the slash). I know it's a detail, but I would double check if we can do something "better"